src/test/java/Homework19.java

```diff
@@ -0,0 +1,70 @@
+package src.test.java;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class Homework19 {
+
+    private WebDriver driver;
+
+    @BeforeTest
+    @Parameters("baseUrl")
+    public void setup(String baseUrl) {
+        System.setProperty("webdriver.chrome.driver", "/path/to/chromedriver");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.get(baseUrl);
+        // Login with your credentials
+        login("username", "password");
+    }
+
+    @AfterTest
+    public void teardown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
```

Comment:
BaseTest — Implements the necessary methods for work with webdriver.

Please, move this code in class BaseTest

![image](https://user-images.githubusercontent.com/104761482/222885590-5fa67e4f-9c55-4ead-839c-fb90301ba4cb.png)